fcglDB.sql

```diff
@@ -0,0 +1,87 @@
+CREATE DATABASE IF NOT EXISTS fcgl;
+
+USE fcgl;
+
+#The condition of an item (example: like new, very good, good...)
+CREATE TABLE IF NOT EXISTS item_condition (
+	item_condition_id INT UNSIGNED NOT NULL AUTO_INCREMENT,
```

Comment:
Indentation is not consistent

Reply:
Yeah, I'm not sure why that happened. I'll try to fix it

Reply:
Fixed